src/xlsx/xlsx_writer.ts

```diff
@@ -138,7 +139,29 @@ function createWorksheets(data: ExcelWorkbookData, construct: XLSXStructure): XL
           )
         );
       }
+    }
+
+    const images = sheet.images;
+    for (const image of images) {
+      const xlsxImageId = convertImageId(image.id);
+      const imageRelId = addRelsToFile(
+        construct.relsFiles,
+        `xl/drawings/_rels/drawing${sheetIndex}.xml.rels`,
+        {
+          target: `../media/image${xlsxImageId}`,
+          type: XLSX_RELATION_TYPE.image,
+        }
+      );
+      drawingRelIds.push(imageRelId);
+      files.push({
+        path: `xl/media/image${xlsxImageId}`,
+        content: image.data.path, // `file.content` stores relative path to avoid unnecessary data transfer
```

Comment:
This needs to be properly documents at least in the readme so lambda users can implement the replacement of files. Right now, one would need to look at the demo data to maybe understand what's going on.

Reply:
> This needs to be properly documents at least in the readme so lambda users can implement the replacement of files. Right now, one would need to look at the demo data to maybe understand what's going on.

@rrahir I saw on the documentation of exporting Excel is still under construction. Has anybody started to work on it?

Reply:
A new starred task is created for doc of exporting: [task 3193306](https://www.odoo.com/web#id=3193306&cids=1&menu_id=4720&action=333&active_id=2328&model=project.task&view_type=form)